pkg/kanx/server.go

```diff
@@ -199,14 +199,14 @@ func NewServer() *Server {
 }
 
 func (s *Server) Serve(ctx context.Context, addr string) error {
-	stopChan := make(chan os.Signal, 1)
-	signal.Notify(stopChan, syscall.SIGTERM, syscall.SIGINT)
+	ctx, can := signal.NotifyContext(ctx, syscall.SIGTERM, syscall.SIGINT)
```

Comment:
nit: make it explicit here `can` => `cancel`. It becomes easier to understand for the human reader.

Reply:
Based on https://go.dev/wiki/CodeReviewComments#variable-names, maybe this should be `c`. The readable scope of the variable is 1 line, so i'd suggest a short variable name

Reply:
Tom, yes, in general variable names should be proportional to the context and so on.
In this case, `cancel` would be a better name for clarity, especially for the casual reviewer that looks at this code.